From 0178e9bf075f2a70a1ad5da8c025f51985de9ee0 Mon Sep 17 00:00:00 2001
From: Chao Zeng <chao.zeng@siemens.com>
Date: Fri, 15 Jan 2021 15:23:02 +0800
Subject: [PATCH 26/27] enable soc device ID driver

This allows Texas Instruments Keystone 3 SoCs to identify
specifics about the SoC in use

Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
---
 configs/iot2050_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/iot2050_defconfig b/configs/iot2050_defconfig
index f2f2496c11..38619f3f7b 100644
--- a/configs/iot2050_defconfig
+++ b/configs/iot2050_defconfig
@@ -121,6 +121,8 @@ CONFIG_DM_RESET=y
 CONFIG_RESET_TI_SCI=y
 CONFIG_DM_SERIAL=y
 CONFIG_SOC_TI=y
+CONFIG_SOC_DEVICE_TI_K3=y
+CONFIG_SOC_DEVICE=y
 CONFIG_TI_PRUSS=y
 CONFIG_SPI=y
 CONFIG_DM_SPI=y
-- 
2.25.1

